pydeseq2/dds.py

```diff
@@ -238,6 +238,21 @@ def deseq2(self):
             # for genes that had outliers replaced
             self.refit()
 
+    @property
+    def normed_counts(self):
+        """
+        Return median-of-ratios normalized counts.
```

Comment:
What do you think about adding a math formula to explain what this median of ratios exactly is, with respect to the original counts?

Reply:
Why not, but I think it would be more relevant to write the formula in `preprocessing.py`, where the code to compute normalisation factors can be found (particularly so as a readable formula won't be a one-liner).

Reply:
OK we can do this, in this case can you add a reference to relevant method in this docstring? In a "See Also" style, so as to point new users to the correct explanation?